ui/components/institutional/confirm-remove-jwt.modal/confirm-remove-jwt-modal.js

```diff
@@ -0,0 +1,124 @@
+import React, { memo, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import Modal from '../../app/modal';
+import CustodyAccountList from '../../../pages/create-account/institutional/connect-custody/account-list';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { removeAccount } from '../../../store/actions';
+import withModalProps from '../../../helpers/higher-order-components/with-modal-props';
+import { Text } from '../../component-library';
+import Box from '../../ui/box';
+import {
+  BorderRadius,
+  DISPLAY,
+  TEXT_ALIGN,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+const ConfirmRemoveJWT = ({
+  custodyAccountDetails,
+  accounts,
+  token,
+  hideModal,
+}) => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const [showMore, setShowMore] = useState(false);
+  const [tokenAccounts, setTokenAccounts] = useState([]);
+
+  useEffect(() => {
+    const tokens = custodyAccountDetails
+      .filter((item) =>
+        accounts.find(
+          (acc) => acc.address.toLowerCase() === item.address.toLowerCase(),
+        ),
+      )
+      .map((item) => ({
+        address: item.address,
+        name: item.name,
+        labels: item.labels,
+        balance: accounts.find(
+          (acc) => acc.address.toLowerCase() === item.address.toLowerCase(),
+        )?.balance,
+        token:
+          item.authDetails?.token ??
+          item.authDetails?.jwt ??
+          item.authDetails?.refreshToken,
+      }))
+      .filter((acc) => acc.token.toLowerCase() === token.address.toLowerCase());
```

Comment:
Question: can we move this filter to line 32 to not look the array one extra time?

Reply:
Done